.gitignore

```diff
@@ -1,12 +1,9 @@
-coverage
```

Comment:
it looks like these have been sorted alphabetically. if we're going to change the order here, can we update to be directories in alphabetical order, followed by files in alphabetical order?